tldw_Server_API/app/core/Sharing/share_audit_service.py

```
@@ -0,0 +1,74 @@
+                event_type=event_type,
+                resource_type=resource_type,
+                resource_id=resource_id,
+                owner_user_id=owner_user_id,
+                actor_user_id=actor_user_id,
+                share_id=share_id,
+                token_id=token_id,
+                metadata=metadata,
+                ip_address=ip_address,
+                user_agent=user_agent,
+            )
+        except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)